server/node/config.js

```diff
@@ -13,10 +14,25 @@ module.exports = {
   country: 'US',
 
   // Store currency.
-  // Note: A few payment methods like iDEAL or SOFORT only work with euros,
-  // so it's a good common denominator to test both Elements and Sources.
   currency: 'eur',
 
+  // Supported payment methods for the store.
+  // Some payment methods support only a subset of currencies.
+  // Make sure to check the docs: https://stripe.com/docs/sources
+  paymentMethods: [
+    // 'ach_credit_transfer', // usd (ACH Credit Transfer payments must be in U.S. Dollars)
+    'alipay', // Can be aud, cad, eur, gbp, hkd, jpy, nzd, sgd, or usd.
```

Comment:
suuuper nit pick but would it look better to remove "Can be" from the alipay and wechat lines to make the comment format very consistent? so just:
```
    'alipay', // aud, cad, eur, gbp, hkd, jpy, nzd, sgd, or usd.
    'wechat', // aud, cad, eur, gbp, hkd, jpy, sgd, or usd.
```
makes the object easier to glance at

Reply:
Good call. I just copy and pasted from the docs.